windows/firefox.admx

```diff
@@ -49,6 +49,14 @@
     </category>
   </categories>
   <policies>
+    <policies>
+      <policy name="AppUpdateURL" class="Both" displayName="$(string.AppUpdateURL)" explainText="$(string.AppUpdateURL_Explain)" key="Software\Policies\Mozilla\Firefox" presentation="$(presentation.AppUpdateURL)" >
+      <parentCategory ref="Firefox" />
+      <supportedOn ref="SUPPORTED_FF60ESR" />
```

Comment:
We'll need to add a new version string to cover this because it's actually 60.2 and Firefox 62.

Reply:
So can I do something like:

<supportedOn ref="SUPPORTED_FF60.2ESR" />
<supportedOn ref="SUPPORTED_FF62" />

?

Is this configuration ESR only? In my eyes this should be ESR only.